pkg/client/results/reader.go

```diff
@@ -49,18 +49,22 @@ const (
 const (
 	// UnknownVersion lets the consumer know if this client can detect the archive version or not.
 	UnknownVersion = "v?.?"
-	versionEight   = "v0.8"
-	versionNine    = "v0.9"
-	versionTen     = "v0.10"
+	VersionEight   = "v0.8"
+	VersionNine    = "v0.9"
+	VersionTen     = "v0.10"
 )
 
 // Reader holds a reader and a version. It uses the version to know
-// where to find certain files within the archive stream.
+// where to find certain files within the archive.
```

Comment:
"certain" is vague and potentially can cause doubt. Can we either identify specifically or else omit?